control-plane/rest/service/src/v0/volumes.rs

```diff
@@ -196,6 +196,24 @@ impl apis::actix_server::Volumes for RestApi {
         Ok(share_uri)
     }
 
+    async fn put_volume_size(
+        Path(volume_id): Path<Uuid>,
+        Body(resize_volume_body): Body<models::ResizeVolumeBody>,
+    ) -> Result<models::Volume, RestError<RestJsonError>> {
+        let volume = client()
+            .resize(
+                &ResizeVolume {
+                    uuid: volume_id.into(),
+                    requested_size: resize_volume_body.size as u64,
+                    capacity_limit: None,
```

Comment:
Is this field not exposed via rest?

Reply:
Please ignore, seems like it's covered in the previous review comments.

Reply:
Not for now, as per this comment:
https://github.com/openebs/mayastor-control-plane/pull/700#discussion_r1431311295